common/src/main/java/com/revenuecat/purchases/common/ReceiptInfo.kt

```diff
@@ -2,16 +2,16 @@ package com.revenuecat.purchases.common
 
 import com.revenuecat.purchases.models.PricingPhase
 import com.revenuecat.purchases.models.StoreProduct
+import com.revenuecat.purchases.models.googleProduct
 
 class ReceiptInfo(
     val productIDs: List<String>,
     val offeringIdentifier: String? = null,
     val purchaseOptionId: String? = null,
     val storeProduct: StoreProduct? = null,
 
-    // TODO BC5 - replace price/currency with making a single PricingPhase for amazon observermode products?
```

Comment:
I think this is okay to delete this comment? 🤔 

The code changed below is almost example what v5.6 used which should still work for Amazon? (Hopefully 😁)

Reply:
i think we can remove it. we'll have to retest alllll the things for amazon anyway...

Reply:
although it is interesting that i noted those were only passed for amazon observer mode.. how did backend figure out price/currency before?

Reply:
ohhh it'd be in the StoreProduct....my brain is mush sorry. either way, i think it's okay to delay amazon stuff for later.

as long as backend knows these values will be coming in for OTP now, and not just amazon observer mode

Reply:
Ahhh yeah, @greenietea is making changes for the backend to expect price and currency for Google Play OTP!

v5.6 did pass up price and currency so this was a breaking change early in in v6 I think? 🤔